include/deal.II/meshworker/dof_info.h

```diff
@@ -272,15 +278,25 @@ namespace MeshWorker
 //----------------------------------------------------------------------//
 
   template <int dim, int spacedim, typename number>
-  DoFInfo<dim,spacedim,number>::DoFInfo(const DoFHandler<dim,spacedim> &dof_handler)
+  DoFInfo<dim, spacedim, number>::DoFInfo(const dealii::DoFHandler<dim, spacedim> &dof_handler)
     :
-    level_cell (false)
+    level_cell(false)
```

Comment:
- astyle: yes, I ran the indentation

- About the test: I see, it probably won't work, I only tried this on a p-uniform meshes so far. I will get back to this.